artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java

```diff
@@ -425,10 +425,10 @@
            format = Message.Format.MESSAGE_FORMAT)
    void broadcastTimeout(int retry, int maxretry);
 
-   @LogMessage(level = Logger.Level.WARN)
-   @Message(id = 212078, value = "Connection factory parameter ignored {0}",
-      format = Message.Format.MESSAGE_FORMAT)
-   void connectionFactoryParameterIgnored(String parameterName);
```

Comment:
Can you just remove this, and call logger.debug directly where this is being called.

We always use logger.debug at the logger on the class.

Reply:
Is it ok to remove the id 212078 or should I keep it in the debug trace ? There is a test looking for it.

Reply:
ok just saw your last comment.